spec/dummy/app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :two_factorable, :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  scope :with_valid_otp_challenge, ->{ where(User.arel_table[:otp_challenge_expires].gt(1.day.ago)) }
```

Comment:
This was an overwrite that we did to fix a bug of his. Maybe just make 1 commit that converts the REAL code to Arel, explains that the old style was something that was put together for only mongo and this AREL syntax should work across all supported AR databases

Reply:
Note that this commit comes BEFORE the test suite to keep this new suite passing on 100% of commits when merged